devices/Scd4x/Scd4x.cs

```diff
@@ -0,0 +1,225 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.I2c;
+using System.Threading;
+using UnitsNet;
+
+namespace Iot.Device.Scd4x
+{
+    /// <summary>
+    /// Humidity and Temperature Sensor Scd4x.
+    /// </summary>
+    public class Scd4x : IDisposable
+    {
+        private const byte CRC8POLYNOMIAL = 0x31;
```

Comment:
we tend to use pascal case for the constants as well

Reply:
done